### PR TITLE
FAQ Error log

### PR DESCRIPTION
**Pull request by dsarkar**
fixes https://github.com/corona-warn-app/cwa-website/issues/1261

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
...
                             "<ol><li>In einem solchen Fall öffnen Sie bitte ein Issue auf GitHub und beschreiben Sie das fehlerhafte Verhalten.</li><li>Starten Sie die Aufzeichnung in der CWA (---LINK oder Screenshot hier: Wie starte ich die Aufzeichnung?---)<br>- Versuchen Sie, das fehlerhafte Verhalten zu reproduzieren bzw. die Fehlermeldung zu erzeugen.<br>- Ist das gelungen, senden Sie den Fehlerbericht an den CWA-Server. Die Daten werden dabei anonymisiert und verschlüsselt. (Bei eigenem Interesse können Sie den Bericht auch lokal auf Ihrem Smartphone abspeichern.)<br>- Als Rückmeldung erhalten Sie eine eindeutige Kennung, die Sie auch in der Liste aller IDs finden <br>- Stoppen Sie die Fehleranalyse.</li><li>Aktualisieren Sie das GitHub-Issue mit Ihrer eindeutigen Kennung vom Server, damit der aufgezeichnete Bericht Ihrem Issue zugeordnet werden kann.</li></ol>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:56]: This is still a placeholder, which should be removed or filled with content:

"---LINK oder Screenshot hier: Wie starte ich die Aufzeichnung?---"

File: src/data/faq.json [hunk 2]
```diff
@@ -51,9 +51,10 @@
...
+                            "<ol><li>If so, please open a issue on GitHub and describe the problematic behavior as detailed as possible.</li><li>Start logging in the CWA (---LINK or screenshot here: How do I start the logging?---).<br>-Try to reproduce the faulty behavior or try to generate the error message.<br>- If this attempt was successful, send the error report to the CWA server. The data will be anonymized and encrypted. (If you are interested, you may also save the report locally on your Smartphone.)<br>- In return you will receive a unique ID, which you can also find in the list of all IDs.<br>- Stop the error report.</li><li>Update the GitHub issue with your unique identifier from the server so that the report can be linked to your issue.</li></ol>",
```
Comment (MikeMcC399) [inline src/data/faq.json:55]: Placeholder should be removed or replaced with content:

"---LINK or screenshot here: How do I start the logging?---"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Placeholder needs work.

**Comment by dsarkar** [comment]
image

image

**Pull request merged**